Makefile

```diff
@@ -1,19 +1,25 @@
-VERSION := $(shell cat VERSION)
-PY2VERSION := $(shell cat PY2VERSION)
+BUILD_VERSION := $(shell cat VERSION)
 
 define build-image
+endef
+
+define push-image
+endef
+
+.PHONY: build
+build:
 	docker pull digitalmarketplace/base
-	docker build --cache-from digitalmarketplace/base -t digitalmarketplace/base -f base${BASE_IMAGE_SUFFIX}.docker .
+	docker build --cache-from digitalmarketplace/base -t digitalmarketplace/base -f base.docker .
 	docker tag digitalmarketplace/base digitalmarketplace/base:${BUILD_VERSION}
 
 	docker build -t digitalmarketplace/base-api -f api.docker .
 	docker tag digitalmarketplace/base-api digitalmarketplace/base-api:${BUILD_VERSION}
 
 	docker build -t digitalmarketplace/base-frontend -f frontend.docker .
 	docker tag digitalmarketplace/base-frontend digitalmarketplace/base-frontend:${BUILD_VERSION}
-endef
 
-define push-image
+.PHONY: push
+push:
 	docker push digitalmarketplace/base:${BUILD_VERSION}
 	if [ "$${BUILD_VERSION:0:1}" -ne "1" ]; then docker push digitalmarketplace/base:latest; fi
```

Comment:
Can we drop these checks as we're only going to building/pushing > version 2.x.x?